code/game/machinery/adv_med.dm

```diff
@@ -554,8 +559,15 @@
 				i_cancer = "Metastatic Tumor:"
 
 		dat += "<tr>"
-		dat += "<td>[i.name]</td><td>N/A</td><td>[i.damage]</td><td>[infection][i_cancer][mech]</td><td></td>"
+		if(i.status & ORGAN_CUT_AWAY)
+			dat += "<td>[i.name]</td><td>-</td><td>-</td><td><font color='red'>Surgically Detached</font></td>"
+		else
+			dat += "<td>[i.name]</td><td>N/A</td><td>[i.damage]</td><td>[infection][i_cancer][mech]</td><td></td>"
 		dat += "</tr>"
+		organs_to_list -= i.type
```

Comment:
since this is a strict type, it will probably fail to work correctly for other species. For example, it would list lungs as missing for plasmamen

although less elegant looking, it can be solved by looping through organs to list and doing an istype comparison, then removing it if true